### DISCUSSION TITLE
Is there a way to use Wxt with Astro (https://astro.build)?

### DISCUSSION THREAD

**Discussion by supfiger** [wxt-dev/wxt#810] [Q&A]
.

**Comment by aklinker1** [comment]
Never looked into astro before, but I doubt it... It seems too focused on websites, but someone who knows more about it's integrations and internals might know more.

If you just need to display markdown content for an FAQ or help page, you can use a vite plugin for that: https://www.npmjs.com/package/vite-plugin-markdown

I should write an example for how to do this.

**Reply by supfiger** [reply]
There is https://github.com/AminoffZ/catonaut repo (Bun + Astro web extension template).
So, Astro can be used to build web extensions. 

But the thing is Wxt is a wrapper over Vite.
So that's why I do not know how to make it work...

**Reply by aklinker1** [reply]
Interesting. Technically, vite is abstracted into a builder interface, so I could implement an astro builder. But unless there's lots of demand for it, I won't be doing that.
